include/CppUTest/UtestMacros.h

```diff
@@ -200,6 +200,29 @@
 #define UNSIGNED_LONGS_EQUAL_LOCATION(expected, actual, text, file, line)\
   { UtestShell::getCurrent()->assertUnsignedLongsEqual((unsigned long)expected, (unsigned long)actual, text, file, line); }
 
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
I still don't really agree (the cases where the user will get the error are the same regardless of whether the `#ifdef` is there or not -- it just changes which error they get and how they discover how to fix it).

But you seem set on this and I'm tired of arguing about it, so I'll remove it in a few minutes.  At which point this whole thread will disappear. :laughing: